onnx/defs/tensor/defs.cc

```diff
@@ -1237,7 +1237,7 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Squeeze_ver11_doc = R"DOC(
+static const char* Squeeze_ver12_doc = R"DOC(
 Remove single-dimensional entries from the shape of a tensor.
```

Comment:
this definition should be updated, given non-one axis value is supported (with no change to the input).